src/actions/topic.actions.js

```diff
@@ -0,0 +1,34 @@
+import { TOPICS_GET_REQUEST_SUCCESS, TOPICS_GET_REQUEST } from '../actions/action_types';
+import { TOPICS_GET_REQUEST_ERROR } from './action_types';
+import TopicService from '../shared/services/TopicService';
+
+export const getTopics = () => {
+    return function (dispatch) {
+        dispatch(topicFetchInit());
+        return TopicService.getSubmittedTopics().then(topics => {
+            dispatch(topicsFetched(topics));
+        }).catch(error => {
+            dispatch(topicsFetchError(error));
```

Comment:
props for error handling 👍

Reply:
it is not reflected anywhere in the app yet - I suppose it would be great to use some Toast/SnackBars to show them!